spec/listformat.html

```diff
@@ -393,10 +411,10 @@ <h1>Properties of Intl.ListFormat Instances</h1>
       <li>[[Locale]] is a String value with the language tag of the locale whose localization is used by the list format styles.</li>
       <li>[[Type]] is one of the String values `"conjunction"`, `"disjunction"`, or `"unit"`, identifying the list of types used.</li>
       <li>[[Style]] is one of the String values `"long"`, `"short"`, or `"narrow"`, identifying the list formatting style used.</li>
-      <li>[[TemplatePair]] is a template for lists of size two. It is expected to contain substrings `"{0}"` and `"{1}"`.</li>
-      <li>[[TemplateStart]] is a template for lists of size greater than two, to be used at the start of the output. It is expected to contain substrings `"{0}"` and `"{1}"`.</li>
-      <li>[[TemplateMiddle]] is a template for lists of size greater than two, to be used in the middle of the output. It is expected to contain substrings `"{0}"` and `"{1}"`.</li>
-      <li>[[TemplateEnd]] is a template for lists of size greater than two, to be used at the end of the output. It is expected to contain substrings `"{0}"` and `"{1}"`.</li>
+      <li>[[TemplatePair]] is a template for lists of size two. It is expected to contain substrings `"{0}"` and `"{1}"` in that order.</li>
```

Comment:
"in that logical order"?

Reply:
What does "logical order" mean in this context?  Maybe the word "order" is not the right word.  I rephrased.